monai/data/iterable_dataset.py

```diff
@@ -149,6 +149,7 @@ class CSVIterableDataset(IterableDataset):
     Args:
         filename: the filename of CSV file to load. it can be a str, URL, path object or file-like object.
             if providing a list of filenames, it will load all the files and join tables.
+        iter: if proving `iter` for stream input directly, skip loading from filename.
```

Comment:
```suggestion
        iter: if provided for stream input directly, skip loading from filename.
```

Reply:
updated in the latest commit.

Thanks.